packages/esm-patient-chart-app/src/visit/visits-widget/active-visits-summary.component.tsx

```diff
@@ -0,0 +1,83 @@
+import { useTranslation } from 'react-i18next';
+import { useVisits } from './visit.resource';
+import VisitSummary from './past-visits-components/visit-summary.component';
+import styles from './visit-detail-overview.scss';
+
+interface ActiveVisitOverviewComponentProps {
+  patientUuid: string;
+}
+
+function ActiveVisitDetailOverviewComponent({ patientUuid }: ActiveVisitOverviewComponentProps) {
+  const { t } = useTranslation();
+  const { visits, error, isLoading, mutateVisits } = useVisits(patientUuid);
+
+  const activeVisits = visits?.filter((visit) => visit.stopDatetime === null);
+
+  if (isLoading) {
+    return (
+      <InlineLoading
+        status="active"
+        iconDescription={t('loading', 'Loading')}
+        description={t('loadingVisit', 'Loading active visits...')}
+      />
+    );
+  }
+
+  if (!!error) {
+    return <ErrorState headerTitle={t('failedToLoadActiveVisits', 'Failed loading active visits')} error={error} />;
+  }
+
+  return (
+    <div className={styles.tabs}>
+      <Tabs>
+        <TabList aria-label="Active visit tabs" contained>
+          {activeVisits?.map((visit, index) => (
+            <Tab label={visit?.visitType?.name} key={index} id={`${index}-id}`}>
+              {visit?.visitType?.name} {t('visit', 'Visit')}
+            </Tab>
+          ))}
+        </TabList>
+        <TabPanels>
+          {activeVisits?.map((visit, index) => (
+            <TabPanel key={index}>
+              {isLoading ? (
+                <InlineLoading description={`${t('loading', 'Loading')} ...`} role="progressbar" />
+              ) : error ? (
+                <ErrorState headerTitle={t('activeVisit', 'Active visit')} error={error} />
+              ) : visit ? (
+                <div className={styles.container}>
+                  <div className={styles.header}>
+                    <div className={styles.visitInfo}>
+                      <div>
+                        <h4 className={styles.visitType}>{visit.visitType.name}</h4>
+                        <div className={styles.displayFlex}>
+                          <h6 className={styles.dateLabel}>{t('start', 'Start')}:</h6>
+                          <span className={styles.date}>{formatDatetime(parseDate(visit.startDatetime))}</span>
+                          {visit.stopDatetime ? (
+                            <>
+                              <h6 className={styles.dateLabel}>{t('end', 'End')}:</h6>
+                              <span className={styles.date}>{formatDatetime(parseDate(visit.stopDatetime))}</span>
+                            </>
+                          ) : null}
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <VisitSummary visit={visit} patientUuid={patientUuid} />
+                </div>
+              ) : (
+                <EmptyState headerTitle={t('visit', 'Visit')} displayText={t('visit', 'Visit')} />
+              )}
```

Comment:
This doesn't make much sense, if there are no visits, then `activeVisits` should be an empty array, hence the tile Tab Panel shouldn't even render 🤔 

Please see if we will fall in the line72?
Thanks!